elasticsearch-dsl/spec/elasticsearch/dsl/search/filters/bool_spec.rb

```diff
@@ -32,6 +32,56 @@
 
   describe '#initialize' do
 
+    context 'when an object instance is provided' do
+
+      let(:search) do
+        described_class.new.must( Elasticsearch::DSL::Search::Queries::Term.new foo: 'bar')
+      end
+
+      it 'executes the block' do
```

Comment:
isn't just the `#must` method called and no block executed?

Reply:
Yes! My fault. If I fix by "calls the method" ok for you ?

Reply:
yes, that makes sense to me. Thank you!

Reply:
Sorry I edited the fix. As the method #to_hash does not call a method but use the stored hash, maybe it's better to write "contains added filter". What do you think ?

Reply:
hm, maybe something like `applies the condition`?

Reply:
Sounds good. I made the fixes.

Reply:
Sounds good. I made the fixes. :)

Reply:
thank you!